int8 STOP=0
int8 FORWARD=1
int8 CWISE=2
int8 CCWISE=3

int8 movement
---
std_msgs/Bool state
